src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataService.java

```diff
@@ -621,22 +610,22 @@ private Object extractChoices(FieldDto fieldDto) {
 
     public Map<String, String> updateTeamByStageAndTexts(UUID caseUUID, UUID stageUUID, String stageType, String teamUUIDKey, String teamNameKey, String[] texts) {
         log.debug("Updating Team by Stage: {} {}", stageUUID, stageType);
-        Map<String, String> dataMap = getCaseData(caseUUID).getDataMap(objectMapper);
+        Map<String, String> dataMap = getCaseData(caseUUID).getDataMap();
         // build the linkValue text string used to search the team link table by converting "text" key to the case's data value
-        String linkValue = null;
+        StringBuilder linkValue = null;
```

Comment:
Intellij suggestion...

Reply:
Not actually quicker for very simple string concatenations, negligable though...